discord/channel.go

```diff
@@ -27,8 +27,52 @@ const (
 	ChannelTypeGuildPrivateThread
 	ChannelTypeGuildStageVoice
 	ChannelTypeGuildDirectory
+	ChannelTypeGuildForum
 )
 
+type ChannelFlags int
+
+const (
+	ChannelFlagsNone   ChannelFlags = 0
+	ChannelFlagsPinned ChannelFlags = 1 << iota
```

Comment:
To make it consistent with other flag naming in the library
```suggestion
	ChannelFlagNone   ChannelFlags = 0
	ChannelFlagPinned ChannelFlags = 1 << iota
```